src/buffer/out/textBuffer.cpp

```diff
@@ -2352,7 +2405,7 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
             // only because we ran out of space.
             if (iRight < cOldColsTotal && !row.WasWrapForced())
             {
-                if (iRight == cOldCursorPos.X && iOldRow == cOldCursorPos.Y)
+                if (!fFoundCursorPos && (iRight == cOldCursorPos.X && iOldRow == cOldCursorPos.Y))

```

Comment:
Unrelated condition change -- scary? I guess as long as the reflow tests still pass...

Reply:
It actually did end up being related. Because we're now also copying the trailing attrs, there was a case where we'd copy the attrs to the end of the row, but then set the new cursor position to the last column of the row the cursor was on. 


(it may not be relevant anymore, but it's not more wrong.)